src/mocks/Trigger.tsx

```diff
@@ -77,7 +78,7 @@ export const getTriggerQuery = (frequency: any) => ({
       trigger: {
         trigger: {
           days: [1, 2],
-          endDate: '2021-03-13',
+          endDate: dayjs('2021-03-13'),
```

Comment:
This is a query mock. It represents the data that will be sent by the backend using a particular query. In this case backend will send back a date string only and not a dayjs object. Please update this.